ui/plugins/auto_classification/controller.js

```diff
@@ -839,10 +839,12 @@ treeherder.controller('ClassificationPluginCtrl', [
 
             // If we have a TextLogSummaryModel then we should have enough data to
             // load this panel
-            var checkLoaded = ThTextLogSummaryModel.get($scope.jobId,
-                                                        {timeout: requestPromise,
-                                                         cache: true});
-            checkLoaded
+            ThTextLogSummaryModel
+                .get($scope.jobId,
+                     {timeout: requestPromise,
+                      cache: true})
+                .then(() => true)
+                .catch(() => false)
                 .then(function(loaded) {
```

Comment:
This whole "retry if not loaded" thing seems pretty silly to me. Why do we want to do this? Shouldn't we just show an error if it fails?

I'm fine with you landing this fix since the code is no worse than before but I think we should change this.